app/controllers/applicants_controller.rb

```diff
@@ -9,7 +9,7 @@ class ApplicantsController < ApplicationController
 '''
   def index
     if params[:session_id]
-      render json: applicants_from_session(params[:session_id])
+      render json: applicants_from_session(params[:session_id], params[:utorid])
```

Comment:
@rathukugan I forgot to add that you should change the authorization for the 4 controllers we edited today. The line is at the start of each controller.  `before_action :tapp_admin` should be changed to:
```
before_action :tapp_admin, exclude: [:index]
before_action :tapp_access, only: [:index]
```
You should also test out the views in Sleepy before merging. To do that, you would need to add yourself as a `TAPP_ADMIN` in the `.env` file. In addition, you need to add yourself in the TAPP UI as a instructor. The option is in the courses view. You should probably ask @reidka for when to test it since it's the default dummy machine for this project and there tend to be always a branch running on it.